parse.c

```diff
@@ -668,15 +669,17 @@ static int __handle_option(const struct fio_option *o, const char *ptr,
 			log_err("not a floating point value: %s\n", ptr);
 			return 1;
 		}
-		if (uf > o->maxfp) {
-			log_err("value out of range: %f"
-				" (range max: %f)\n", uf, o->maxfp);
-			return 1;
-		}
-		if (uf < o->minfp) {
-			log_err("value out of range: %f"
-				" (range min: %f)\n", uf, o->minfp);
-			return 1;
+		if (o->minfp || o->maxfp) {
```

Comment:
So if either of the fp limits is set you check both of them - is this correct or should the guard be rolled into the sub conditions?

Reply:
Yes, if any of the fp limits has been set both are checked. So if someone only wants to set one limit the other should be set to DBL_MIN or DBL_MAX as appropriate.

Reply:
OK that makes sense.